### ISSUE TITLE
Vault and signature is corrupted in OC 1.0.2

### ISSUE THREAD

**Issue by junzhli** [acidanthera/bugtracker#2443]
Hello team,

I build my own OC with vault and signature enforced.
I build its vault.plist and vault.sig as always as I did before, and it can't work properly since 1.0.2
It works seamlessly before 1.0.2
```
OCST: Invalid vault signature
OC: Failed to open root FS - Security Violation!
```
![20241104_223003](https://github.com/user-attachments/assets/0d6e061e-c75b-457f-9d52-5ffc8ffdc040)

Here's the error occurred as shown in the screenshot. It seems like something is wrong here?
Currently, I can only fall back to using "basic" option to enforce integrity validation with less secure level
Thanks!!

**Comment by junzhli** [comment]
I notice the issue is gone when swapped to the debug build of opencore, it seems like there's a trouble in this release build

**Comment by Gobias64** [comment]
I'm getting the same error with Vault set to Secure in OC 1.0.2. I used sign.command from the latest release to create vault.plist and vault.sig like I always do.

**Comment by mikebeaton** [comment]
@junzhli @Gobias64 - The updated `sign.command` script here https://github.com/acidanthera/OpenCorePkg/tree/master/Utilities/CreateVault can be used to correctly sign your 1.0.2 release, if you want to try that.

**Comment by Gobias64** [comment]
> @junzhli @Gobias64 - The updated `sign.command` script here https://github.com/acidanthera/OpenCorePkg/tree/master/Utilities/CreateVault can be used to correctly sign your 1.0.2 release, if you want to try that.

Thank you! I saw the fix in the 1.0.3 release notes as well. I'll try it with the new release when I get a chance.
